### PR TITLE
Add json field to operations export

### PR DESCRIPTION
**Pull request by chowbao**
<details>
  <summary>PR Checklist</summary>

### PR Structure

- [ ] This PR has reasonably narrow scope (if not, break it down into smaller PRs).
- [ ] This PR avoids mixing refactoring changes with feature changes (split into two PRs
      otherwise).
- [ ] This PR's title starts with the jira ticket associated with the PR.

### Thoroughness

- [ ] This PR adds tests for the most critical parts of the new functionality or fixes.
- [ ] I've updated the README with the added features, breaking changes, new instructions on how to use the repository. I updated the description of the fuction with the changes that were made.

### Release planning

- [ ] I've decided if this PR requires a new major/minor/patch version accordingly to
    [semver](https://semver.org/), and I've changed the name of the BRANCH to release/_ , feature/_ or patch/\* .
</details>

### What

Add JSON blob of `operation.details`

### Why

To leverage BigQuery JSON data type as well as aid downstream dbt models

### Known limitations

* Need to test in test-hubble

### CODE REVIEW

File: internal/transform/operation.go [hunk 1]
```diff
@@ -83,17 +83,18 @@ func TransformOperation(operation xdr.Operation, operationIndex int32, transacti
...
+		Type:                 outputOperationType,
+		TypeString:           outputOperationTypeString,
+		TransactionID:        outputTransactionID,
+		OperationID:          outputOperationID,
+		OperationDetails:     outputDetails,
+		ClosedAt:             outputCloseTime,
+		OperationResultCode:  outputOperationResultCode,
+		OperationTraceCode:   outputOperationTraceCode,
+		LedgerSequence:       outputLedgerSequence,
+		OperationDetailsJSON: outputDetails,
```
Comment (amishas157) [inline internal/transform/operation.go:97]: What is the real difference between OperationDetails and OperationDetailsJSON

File: internal/transform/operation_test.go
Comment (sydneynotthecity) [inline internal/transform/operation_test.go:1]: This is gonna be obnoxious to maintain. I'll start communicating the changes to external partners so we can work on dropping the extra `OperationDetailsJSON`

**Review by sydneynotthecity — APPROVED** [review]
LGTM

File: internal/transform/operation.go (hunk 1 above)
Comment (amishas157) [reply internal/transform/operation.go:97]: I see. Are you going to drop individual fields subsequently and keep only the json blob?

File: internal/transform/operation_test.go
Comment (chowbao) [reply internal/transform/operation_test.go:1]: Yeah. Our unit tests are a pain in general unfortunately

File: internal/transform/operation.go (hunk 1 above)
Comment (chowbao) [reply internal/transform/operation.go:97]: We could drop the `details` record eventually after we talk with partners and they move off of it. Oh I guess we would also need to move off of it as well

So yes in theory the end result would be
* drop `details`
* rename `details_json` to `details`

**Review by amishas157 — APPROVED** [review]

**Pull request merged**